packages/@aws-cdk/aws-cloudformation/lib/pipeline-actions.ts

```diff
@@ -33,6 +33,17 @@ export interface PipelineCloudFormationActionProps extends codepipeline.CommonAc
    * @default Automatically generated artifact name.
    */
   outputArtifactName?: string;
+
+  /**
+   * The region the given Action resides in.
+   * Note that there are some scaffold resources that need to be created before a Pipeline can be cross-region.
```

Comment:
Instead of "scaffold resources" just write "replication buckets"

Reply:
Changed.